cmatrix.c

```diff
@@ -85,6 +85,7 @@ typedef struct cmatrix {
 int console = 0;
 int xwindow = 0;
 int lock = 0;
```

Comment:
Maybe change type to `bool` from `stdbool.h` ?

Reply:
Okay